plasmapy/plasma/sources/openpmd_hdf5.py

```diff
@@ -148,8 +148,7 @@ def is_datasource_for(cls, **kwargs):
         hdf5 = kwargs.get("hdf5")
         openPMD = kwargs.get("openPMD")
 
-        isfile = os.path.isfile(hdf5)
-        if not isfile:
+        if not Path(hdf5).is_file():
```

Comment:
This line of code is not covered by tests.

Reply:
Ah, missed that comment. Let me still investigate this before merging .